packages/core/src/common/classes.ts

```diff
@@ -193,7 +193,12 @@ export const FORM_HELPER_TEXT = `${NS}-form-helper-text`;
 export const FORM_GROUP_SUB_LABEL = `${NS}-form-group-sub-label`;
 
 export const MENU = `${NS}-menu`;
+export const MENU_NO_PADDING = `${MENU}-no-padding`;
```

Comment:
Used to allow MenuSection to handle their own padding thus having a divider that goes edge to edge.